dev-requirements.txt

```diff
@@ -1,4 +1,17 @@
-git+https://github.com/dbt-labs/dbt-redshift.git@c35865b
-git+https://github.com/dbt-labs/dbt-snowflake.git@a017382
-git+https://github.com/dbt-labs/dbt-bigquery.git@d52222e
-git+https://github.com/dbt-labs/dbt-core.git@0db634d#egg=dbt-core&subdirectory=core
\ No newline at end of file
+pytest
+pytest-dotenv
+
+# Bleeding edge
+# git+https://github.com/dbt-labs/dbt-core.git@main#egg=dbt-tests-adapter&subdirectory=tests/adapter
+# git+https://github.com/dbt-labs/dbt-core.git@main#egg=dbt-core&subdirectory=core
+# git+https://github.com/dbt-labs/dbt-core.git@main#egg=dbt-postgres&subdirectory=plugins/postgres
+# git+https://github.com/dbt-labs/dbt-redshift.git
+# git+https://github.com/dbt-labs/dbt-snowflake.git
+# git+https://github.com/dbt-labs/dbt-bigquery.git
+
+# Most recent release candidates
+dbt-tests-adapter==1.2.0rc1
+dbt-core==1.2.0rc1
+dbt-redshift==1.2.0rc1
+dbt-snowflake==1.2.0rc1
+dbt-bigquery==1.2.0rc1
```

Comment:
How will these stay up to date over time?

Reply:
We'll manually update these based on the version that we want to test. Right now its the RC candidate but we'll update it to v1.2 once it is no longer RC.